src/test/java/uk/gov/hmcts/sptribs/ciccase/model/access/DefaultAccessTest.java

```diff
@@ -27,23 +27,27 @@ void shouldGrantDefaultAccess() {
         final SetMultimap<HasRole, Permission> grants = new DefaultAccess().getGrants();
         //Then
         assertThat(grants)
-            .hasSize(22)
+            .hasSize(26)
             .contains(
                 entry(CREATOR, C),
                 entry(CREATOR, R),
                 entry(CREATOR, U),
                 entry(ST_CIC_CASEWORKER, C),
                 entry(ST_CIC_CASEWORKER, R),
                 entry(ST_CIC_CASEWORKER, U),
+                entry(ST_CIC_CASEWORKER, D),
```

Comment:
This needs to be modified based on the action taken as per DefaultAccess.java comment.

Reply:
This change is not needed for Case flags hence reverted.